openedx/features/pakx/lms/overrides/views.py

```diff
@@ -432,54 +463,105 @@ def post(self, request):
                 self.request,
                 _(u'Thank you for contacting us! Our team will get in touch with you soon')
             )
-            return HttpResponseRedirect(self.success_redirect)
+            context = self.get_context_data(request=request)
+            return render_to_response(self.template_name, context)
 
-        context = self.get_context_data()
+        context = self.get_context_data(request=request)
         context['form'] = form
         return render_to_response(self.template_name, context)
 
 
 class PartnerWithUsView(AboutUsView):
     """
-    View for partner-with-us page.
+    View for partner with us page.
     """
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.email_subject = 'Partner with Us Form Data'
 
-    success_redirect = '/partner-with-us/'
     template_name = "overrides/partner_with_us.html"
 
 
-class BusinessView(AboutUsView):
+# class BusinessView(AboutUsView):
```

Comment:
Why have we commented this out?

Reply:
Currently, we are using the W.E demo page as the business page, commented the view just in case we roll back to the previous version. 
@maneeshatejani1 please add a comment there

Reply:
check 'openedx/features/pakx/lms/overrides/views.py' in this PR https://github.com/PakistanX/edx-platform/pull/186/files
![image](https://user-images.githubusercontent.com/79001283/144421291-0f36fb52-b07d-4b5b-b8a4-e2b675c5c38d.png)

Reply:
@mubtadaali I believe you did this change

Reply:
it was done in PR 186 by @mubtadaali and the class was later redefined but I thought maybe it was purposely commented for later need. let me know if need to remove that

Reply:
@mubtadaali Why commented the view then?
Simply remove the respective URL?

Reply:
currently, two URLs are pointing to W.E demo view, business and we-demo URL. I believe we can remove that commented view now.